exporter/stackdriverexporter/stackdriver.go

```diff
@@ -108,16 +115,16 @@ func (se *stackdriverExporter) pushMetricsData(ctx context.Context, md consumerd
 // pushTraceData is a wrapper method on StackdriverExporter.PushSpans
 func (se *stackdriverExporter) pushTraceData(ctx context.Context, td consumerdata.TraceData) (int, error) {
 	var errs []error
-	droppedSpans := 0
+	var goodSpans []*tracepb.Span
 	for _, span := range td.Spans {
 		sd, err := spandatatranslator.ProtoSpanToOCSpanData(span)
 		if err == nil {
 			se.exporter.ExportSpan(sd)
+			goodSpans = append(goodSpans, span)
```

Comment:
Nit: Looks like the only purpose of `goodSpans` is to count the number of spans that succeeded translation. It would be more efficient to use a simple int counter instead of the slice to count this.

Reply:
Sounds good, updated to int for now.